package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "tsp-toolkit",
     "version": "0.17.0",
-    "lockfileVersion": 3,
+    "lockfileVersion": 2,
```

Comment:
What version of npm are you using? I currently have 10.5.0 (packaged with nodejs 21.7.3). You might need to update.